bin/node/src/chain_spec/commands.rs

```diff
@@ -0,0 +1,111 @@
+use std::{io::Write, path::PathBuf};
+
+use sc_cli::{
+    clap::{self, Parser},
+    Error, KeystoreParams,
+};
+use sc_service::config::BasePath;
+
+use crate::chain_spec::{
+    builder::build_chain_spec_json,
+    cli::ChainSpecParams,
+    keystore::{
+        create_abft_backup_dir, create_account_session_keys, create_p2p_key, open_keystore,
+    },
+    AlephNodeChainSpec, DEFAULT_BACKUP_FOLDER,
+};
+
+/// This command generates session keys and libp2p key for all input accounts.
+/// Those keys are written to the keystore in a given base path. It also generates a chainspec.
+#[derive(Debug, Parser)]
+pub struct BootstrapChainCmd {
```

Comment:
this makes me just slightly uncomfortable, that bootstrapping chain (in particular session keys generation, keystore preparation) lives in the `chain_spec` module, which should be somehow independent... but as far as I understand, we would have to artificially split the `BootstrapChainCmd::run` body into two distinct commands, losing a convenient way to prepare local environment with a single instruction :[

I would opt for a better name for the `chain_spec` module (soon-to-be-a-crate), but feel free to ignore this comment.

Reply:
Excellent point, this should be named differently.  tried to make two commands, but code we have now is convenient  to do as as single command, as we need to read session keys from the storage to produce chainspec.